spec/controllers/curation_concerns/file_sets_controller_spec.rb

```diff
@@ -116,22 +116,6 @@
           expect(assigns[:file_set]).to eq file_set
         end
 
-        it 'adds a new groups and users' do
```

Comment:
Setting groups and users is not something you can do in curation_concerns. I think this was from Sufia.